generation/classifier_free_guidance/2d_ddpm_classifier_free_guidance_tutorial.ipynb

```diff
@@ -0,0 +1,959 @@
+{
```

Comment:
<p>It seems that the loss is not decreasing. Is this expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>I think the issue here is that the LDM loss goes down at the start (somehow not captured because it probably goes down quickly within the first epoch). Other than that, it's a noisy loss, which causes it to look like this sometimes even when the model's learning. </p>